qiita_db/metadata_template/sample_template.py

```diff
@@ -252,3 +252,59 @@ def generate_files(self):
             # generating all new QIIME mapping files
             for pt_id in Study(self._id).prep_templates():
                 PrepTemplate(pt_id).generate_files()
+
+    @property
+    def ebi_sample_accessions(self):
+        """The EBI sample accessions for the samples in the sample template
+
+        Returns
+        -------
+        dict of {str: str}
+            The EBI sample accession numbers keyed by sample id
+        """
+        return self._get_accession_numbers('ebi_sample_accession')
+
+    @ebi_sample_accessions.setter
+    def ebi_sample_accessions(self, value):
+        """Sets the EBI sample accessions
+
+        Parameters
+        ----------
+        values : dict of {str: str}
+            The EBI sample accessions, keyed by sample id
+
+        Raises
+        ------
+        QiitaDBError
+            If value tries to change the accession number of a sample that
```

Comment:
What about of something along the lines of:

\`If a sample in \`value\` already has an accession number\`.

Something similar could be done with the rest of the docstrings.